src/features/search/pages/modals/VenueModal/useVenueModal.native.test.tsx

```diff
@@ -21,18 +24,21 @@ jest.mock('features/search/context/SearchWrapper', () => ({
 }))
 
 const dismissMyModal: VoidFunction = jest.fn()
```

Comment:
je trouve que ça simplifirais la lecture comme ça :

```suggestion
const dismissModal: VoidFunction = jest.fn()
```

car on pourrait simplifier les `{ dismissModal: dismissMyModal }` en `{ dismissModal }`

Reply:
je vais le changer, il me semble que lors de ma première écriture je "devais" ne pas mettre le même nom...